integration/util.go

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) 2023 Contributors to the Eclipse Foundation
+//
+// See the NOTICE file(s) distributed with this work for additional
+// information regarding copyright ownership.
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License 2.0 which is available at
+// https://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+// which is available at https://www.apache.org/licenses/LICENSE-2.0.
+//
+// SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+
+package integration
+
+import (
+	"crypto/sha256"
+	"encoding/base64"
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/eclipse-kanto/kanto/integration/util"
+)
+
+func getBootstrapConfigStruct(path string) (util.BootstrapConfiguration, error) {
```

Comment:
you can use pointer here in order to return `nil, err` on error.
```suggestion
func getBootstrapConfigStruct(path string) (*util.BootstrapConfiguration, error) {
```